qa/rpc-tests/wallet-dump.py

```diff
@@ -61,7 +61,11 @@ def __init__(self):
         self.extra_args = [["-keypool=90"]]
 
     def setup_network(self, split=False):
-        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir, self.extra_args)
+        # Use long, 15-minute timeout because the initial getnewaddress RPC can
```

Comment:
The whole wallet dump test should run in less than 15 seconds. I don't see why a 15 *minute* timeout is necessary.

From travis (confirmed locally):
```
wallet-dump.py                 | True   | 12 s
```